frontend/src/pages/projects/screens/detail/workspaces/useWorkspaceImage.ts

```diff
@@ -0,0 +1,35 @@
+import { getDescriptionForTag, getImageTagByContainer } from '../../../../../utilities/imageUtils';
+import { useWatchImages } from '../../../../../utilities/useWatchImages';
+import { NameVersionPair, NotebookContainer } from '../../../../../types';
+import { NotebookKind } from '../../../../../k8sTypes';
+
+export type WorkspaceImage = {
+  imageName: string;
+  tagSoftware: string;
+  packages: NameVersionPair[];
+};
+
+const useWorkspaceImage = (
+  notebook: NotebookKind,
+): [workspaceImage: WorkspaceImage | null, loaded: boolean] => {
+  const { images, loaded } = useWatchImages();
```

Comment:
We should probably use pass-through API here, refer to the hook I created on the spawner page.